core/migrations/versions/cc1d1143315f_create_slack_tables.py

```diff
@@ -23,6 +23,7 @@ def upgrade() -> None:
     sa.Column('message_id', sa.String(50), nullable=False),
     sa.Column('channel_id', sa.String(50)),
     sa.Column('user_id', sa.String(50)),
+    sa.Column('team_id', sa.String(50)),
```

Comment:
should this be project_id? or internal_project_id?

Reply:
The info from slack says team.